offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/product/archive/ArchiveProductView.groovy

```diff
@@ -0,0 +1,14 @@
+package life.qbic.portal.offermanager.components.product.archive
+
+import com.vaadin.ui.HorizontalLayout
+
+/**
+ * <h1><short description></h1>
+ * <br>
+ * <p><detailed description></p>
+ *
```

Comment:
Missing Description 👀

Reply:
👀

Reply:
I removed this file since I found that this use case does not necessarily need a view, a button is sufficient